.github/workflows/artifacts.yml

```diff
@@ -57,12 +60,30 @@ jobs:
           tag: ${{ github.ref }}
           overwrite: true
 
-      - name: Upload Linux
+      - name: Upload Linux x64
         uses: svenstaro/upload-release-action@v1-release
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: ffi/node/libzkgroup.so
-          asset_name: libzkgroup.so
+          file: ffi/node/libzkgroup-x64.so
+          asset_name: libzkgroup-x64.so
+          tag: ${{ github.ref }}
+          overwrite: true
+
+      - name: Upload Linux ia32
+        uses: svenstaro/upload-release-action@v1-release
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: ffi/node/libzkgroup-ia32.so
+          asset_name: libzkgroup-ia32.so
+          tag: ${{ github.ref }}
+          overwrite: true
+
+      - name: Upload Linux arm64
+        uses: svenstaro/upload-release-action@v1-release
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: ffi/node/libzkgroup-arm64.so
+          asset_name: libzkgroup-arm64.so
           tag: ${{ github.ref }}
           overwrite: true
```

Comment:
Let's use `file_glob: true` for these and then we can have a single upload action for all the Linux ones.